src/annotation/app.py

```diff
@@ -85,6 +87,30 @@ def post_annotation():
     return jsonify({"message": "Annotation posted", "id": cur.lastrowid})
 
 
+@app.route("/replace_annotation/<int:anno_id>", methods=['POST'])
```

Comment:
`POST {new data} /replace_annotation/{id}` seems a bit unnatural (why not `PUT {new data} /annotations/{id}`)?

Reply:
Thanks for reviewing. Yes, if you don't mind cleaning up the API that would be great.  I'll go ahead and merge.